lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4404,6 +4404,9 @@ namespace {
     AccessorDecl *tryCreateConstexprAccessor(const clang::VarDecl *clangVar,
                                              VarDecl *swiftVar) {
       assert(clangVar->isConstexpr());
+      auto *Eval = clangVar->ensureEvaluatedStmt();
+      if (cast<clang::Expr>(Eval->Value)->isValueDependent())
```

Comment:
Is it possible for `Value` to be null?

Reply:
I don't think so because there is an assertion at the top to check if it will be a constexpr or not.
A non malformed expression would always have a RHS value.